components/UI/iconsComponents/icons/verifiedVariantIcon.tsx

```
@@ -0,0 +1,31 @@
+  width,
+  color,
+}) => {
+  return (
+    <svg
+      width={width}
+      height="12"
+      viewBox="0 0 13 12"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g clip-path="url(#clip0_4991_3904)">
```

ESLint

[error] Unknown property 'clip-path' found, use 'clipPath' instead

(react/no-unknown-property)